src/fragmentTypes.json

```diff
@@ -0,0 +1,30 @@
+{
+  "__schema": {
+    "types": [
+      {
+        "kind": "UNION",
+        "name": "SearchResultItem",
+        "possibleTypes": [
+          {
+            "name": "Issue"
+          },
+          {
+            "name": "PullRequest"
+          },
+          {
+            "name": "Repository"
+          },
+          {
+            "name": "User"
+          },
+          {
+            "name": "Organization"
+          },
+          {
+            "name": "MarketplaceListing"
+          }
+        ]
```

Comment:
I'm trying to understand this file. Why do we have some many possible types listed here when we don't care about `MarketplaceListing`, `User`, etc.. Maybe if you can link me to some documentation online, it will help me understand.

Reply:
I do it based on this guide. https://www.apollographql.com/docs/react/advanced/fragments.html#fragment-matcher but initially it was 1000+ lines, then I start reducing it to only have UNION that we use.